src/bentoml/_internal/utils/transformers.py

```diff
@@ -0,0 +1,43 @@
+from __future__ import annotations
```

Comment:
Oh we already have an utils for framework under `bentoml/_internal/frameworks/utils`. You can move these function there.